packages/web/src/scss/components/CheckboxField/_CheckboxField.scss

```diff
@@ -64,7 +69,8 @@
     @include form-fields-tools.label-disabled();
 }
 
-.CheckboxField--disabled .CheckboxField__message {
+.CheckboxField--disabled .CheckboxField__message,
+.CheckboxField--disabled .CheckboxField__helperText {
     @include form-fields-tools.message-disabled();
```

Comment:
also this mixin should be named better to cover both message and helpertext

Reply:
So please give me a name suggestion, i will rename it.

Reply:
fixed